scripts/perf/README.md

```diff
@@ -0,0 +1,18 @@
+# Perf runner
+
+## Requirements
+
+* Linux
+
+## Generating a flamegraph with a 1GB file download, 0B upload
```

Comment:
Line 7 and line 13 are headings, but I think you could make them both paragraphs under one heading. 
```suggestion
## Generating a Flamegraph
To generate a flamegraph with a...
```
If you wanted to put them under two headings I would rather you did something like "## Default Flamegraph" and "## Customized Flamegraph"
Also I am not clear on what a 1GB file download, 0B upload is, could you elaborate on what that means?